sonar-apex-plugin/pom.xml

```diff
@@ -37,6 +37,26 @@
             <artifactId>apex-check</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
+            <version>2.6.2</version>
+            <type>jar</type>
+        </dependency>
+        <dependency>
+            <groupId>org.easytesting</groupId>
+            <artifactId>fest-assert</artifactId>
+            <version>1.4</version>
+            <scope>test</scope>
+            <type>jar</type>
+        </dependency>
+        <dependency>
+            <groupId>org.easytesting</groupId>
```

Comment:
ApexrulesDefinitionTest for assertions of test_invalid_checks which btw is beeing renamed